internal/controllers/clusterextension_controller.go

```diff
@@ -572,24 +571,24 @@ func (r *ClusterExtensionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	controller, err := ctrl.NewControllerManagedBy(mgr).
 		For(&ocv1alpha1.ClusterExtension{}).
 		Watches(&catalogd.ClusterCatalog{},
-			crhandler.EnqueueRequestsFromMapFunc(clusterExtensionRequestsForCatalog(mgr.GetClient(), mgr.GetLogger()))).
-		WithEventFilter(predicate.Funcs{
-			UpdateFunc: func(ue event.UpdateEvent) bool {
-				oldObject, isOldCatalog := ue.ObjectOld.(*catalogd.ClusterCatalog)
-				newObject, isNewCatalog := ue.ObjectNew.(*catalogd.ClusterCatalog)
-
-				if !isOldCatalog || !isNewCatalog {
-					return true
-				}
+			crhandler.EnqueueRequestsFromMapFunc(clusterExtensionRequestsForCatalog(mgr.GetClient(), mgr.GetLogger())),
+			builder.WithPredicates(predicate.Funcs{
```

Comment:
a tab walks into a bar sits down. The barkeep asks him, why don't space?

Reply:
😄 Yeah, the formatting looks weird because the predicate needs to go with the ClusterCatalog watch, and it means it all gets indented by an extra tab.